SPECS/glibc/glibc.spec

```diff
@@ -16,7 +16,7 @@ URL:            https://www.gnu.org/software/libc
 Source0:        https://ftp.gnu.org/gnu/glibc/%{name}-%{version}.tar.xz
 Source1:        locale-gen.sh
 Source2:        locale-gen.conf
-Patch0:         https://www.linuxfromscratch.org/patches/downloads/glibc/glibc-2.35-fhs-1.patch
+Patch0:         https://www.linuxfromscratch.org/patches/downloads/glibc/glibc-2.38-fhs-1.patch
 # Only applicable on ARMv7 targets.
 Patch1:         CVE-2020-6096.nopatch
```

Comment:
I don't think Mariner needs this patch since ARMv7 is 32 bit only.

Reply:
The patch file fixes references to /var/db (https://www.linuxfromscratch.org/patches/lfs/12.0/glibc-2.38-fhs-1.patch)
The comment here is for the .nopatch file below (not changed in this PR)